src/fentry.bpf.c

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2021 Sartura */
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+char LICENSE[] SEC("license") = "Dual BSD/GPL";
+
+SEC("fentry/bprm_execve")
+int BPF_PROG(bprm_execve, struct linux_binprm *bprm, int fd, struct filename *filename, int flags)
+{
+	pid_t pid;
```

Comment:
nit: move it below for "reverse Christmas tree" effect, preferred style in net/bpf kernel trees, thanks!